cli/src/diff_util.rs

```diff
@@ -1127,6 +1150,16 @@ pub fn show_git_diff(
                         writeln!(formatter, "index {left_hash}..{right_hash}")?;
                     }
                     (Some(left_mode), Some(right_mode)) => {
+                        if left_path != right_path {
+                            let operation = if to_tree.path_value(&left_path).unwrap().is_absent() {
```

Comment:
nit: `path_value(..)?`

The return type of this closure will have to be `Result<(), DiffRenderError>`.

Reply:
Done